hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutAllOperation.java

```diff
@@ -68,6 +68,14 @@ public PutAllOperation(String name, MapEntries mapEntries, boolean initialLoad)
     }
 
     @Override
+    public void beforeRun() throws Exception {
+        super.beforeRun();
+        if (shouldWanReplicate()) {
```

Comment:
What is the diff between `mapContainer.isWanReplicationEnabled()` and `shouldWanReplicate`?